openshift/system/Dockerfile

```diff
@@ -78,6 +80,9 @@ RUN git config --global --add safe.directory '*'
 RUN ln -s openshift/system/entrypoint.sh . \
     && ln -s /opt/system/openshift/system/contrib/container-entrypoint /usr/local/bin/
 
+RUN rm -rf vendor/oracle/ test/ spec/ features/ lib/proxy \
+    && cp config/oracle/*.ini /etc/
```

Comment:
some other stuff I saw us delete in production build so lets delete them here as well